SpatialGDK/Source/SpatialGDKTests/Examples/SpatialGDKTestGuidelines.h

```diff
@@ -0,0 +1,56 @@
+/*
+0. How to run tests:
+	- Tests can be run via Session Frontend in UE4 Editor
+
+	- Tests can be run via command line
+
+1. Folder structure
+	- Tests are located in a separate SpatialGDKTests module.
+	So for every component that needs to be tested - it has to be exposed via corresponding macro.
+	E.g. `class FRPCContainer` -> `class SPATIALGDK_API FRPCContainer`, to make FRPCContainer testable.
+	WThe macro is different in each module SPATIALGDK_API, SPATIALGDKSERVICES_API etc.
+
+	- The folder structure inside SpatialGDKTests should resemble the folder structure for components being tested.
+	E.g. If file tested is
+	`UnrealGDK\SpatialGDK\Source\SpatialGDK\Public\Utils\RPCContainer.cpp`,
+	then the corresponding test should be located in
+	`UnrealGDK\SpatialGDK\Source\SpatialGDKTests\SpatialGDK\Utils\RPCContainer\RPCContainerTest.cpp`.
+	There should be a folder with the same name as component tested. It should include the test and all the supporting files.
+
+2. Test definitions (check TestDefinitions.h for more info)
+    - We have defined 3 types of Macro to be used when writing tests:
+	(https://docs.unrealengine.com/en-US/Programming/Automation/TechnicalGuide/index.html has more information)
+	TEST - the most common simple test
```

Comment:
Might be better to expand on this or just link to the docs. This doesn't really tell me anything.

Reply:
I've linked UE4 docs. Can expand though if needed.

Reply:
I mean that "the most common simple test" doesn't tell me anything.

Reply:
You're right, I've updated it to `TEST - a simple test, that should be used if it's one of a kind (i.e. it's body can't be reused, otherwise use COMPLEX_TEST), and if doesn't rely on background threads doing the computation (otherwise use LATENT_COMMANDs).`